### PR TITLE
Ana/fix the weird CSS bug with ActionModal

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Fixes this:

image

Now looks like this (normal):

image

I know, it is a hacky fix, but it works

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f63a9c3d3538f247ed956d8--lunieio.netlify.app

File: app/public/index.html [hunk 1]
```diff
@@ -49,6 +49,10 @@
   <link rel="icon" type="image/png" href="img/icons/favicon-96x96.png" />
 
   <style>
+    html {
+      position: unset !important;
```
Comment (faboweb) [inline app/public/index.html:53]: interesting. does it get overwritten somewhere?

**Review by jbibla — CHANGES_REQUESTED** [review]
no more "hacky" fixes. where is the css coming from that is causing this issue?

**Comment by Bitcoinera** [comment]
> no more "hacky" fixes. where is the css coming from that is causing this issue?

Good question. Did you reproduce? It is very strange

File: app/public/index.html (hunk 1 above)
Comment (Bitcoinera) [reply app/public/index.html:53]: yes, strangely

**Comment by jbibla** [comment]
i have not been able to but i have seen it a couple times and seen you guys share it before.

**Comment by Bitcoinera** [comment]
> i have not been able to but i have seen it a couple times and seen you guys share it before.

Can you please try to reproduce and see where that is coming from? I have no clue

**Comment by Bitcoinera** [comment]
OK, so Jordan cannot reproduce this on his Mac. I will try on my Windows the next day and see if it is bug only for Linux or what is the issue here

**Comment by faboweb** [comment]
> how do you reproduce it?

scroll down to the bottom then open the action modal

**Comment by Bitcoinera** [comment]
> scroll down to the bottom then open the action modal

Could you reproduce it on your Windows?

**Comment by faboweb** [comment]
it is the `no-scroll` plugin. let's remove it.

**Comment by Bitcoinera** [comment]
> it is the `no-scroll` plugin. let's remove it.

OK, on it

**Comment by faboweb** [comment]
> AppHeader

doesn't use the plugin

**Pull request merged**